src/collection-api/routes/metadata.js

```diff
@@ -20,107 +20,146 @@ const PACKAGE_JSON_PATH = '../../../package.json';
  *       type: object
  *       description: Collection metadata
  *       additionalProperties: false
+ *       required:
+ *         - id
+ *         - name
+ *         - tagline
+ *         - languages
+ *         - jurisdictions
  *       properties:
  *         id:
  *           type: string
- *           description: Unique identifier of the collection
+ *           description: Unique identifier derived from name (acronyms, dash-separated).
+ *           example: pga
```

Comment:
Why do we take `pga` as an example, which is a third-party that we don't control, instead of `demo` that is here to that end?

Reply:
I've simply copied the examples from our current documentation, but you're right.